test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -411,7 +426,7 @@ def setUp(self):
     "exclude_vntypes"       : [],
     "run_only_on_hosts"     : [],
     "periodic_resc_update"  : false,
-    "vnode_per_numa_node"   : true,
+    "vnode_per_numa_node"   : false,
```

Comment:
Why did this change to false?

Reply:
It changed to false since the only other test that uses this configuration is test_cgroup_prefix_and_devices, which doesn't care if there is a vnode or not.